packages/loader/container-loader/package.json

```diff
@@ -111,6 +111,10 @@
     "version": "2.0.0-internal.2.3.0",
     "baselineRange": ">=2.0.0-internal.2.2.0 <2.0.0-internal.2.3.0",
     "baselineVersion": "2.0.0-internal.2.2.0",
-    "broken": {}
+    "broken": {
+      "ClassDeclaration_Container": {
+        "forwardCompat": false
```

Comment:
This PR is targeting main. Can we make the new method on the Container class optional to make this non-breaking? Or should this be on next?

Reply:
I thought it was already optional but I see that's not the case, I'll look into that.

Reply:
The interface property is optional, but the getter on the class isn't (it can return undefined, but the getter itself is required).